app/client/components/svgs/creditCardIcon.tsx

```diff
@@ -0,0 +1,16 @@
+import { palette } from "@guardian/src-foundations";
```

Comment:
Hey, I didn't use the design systems credit card icon because of it's use within the left hand nav that requires it not to have internal svg padding as the deign system icons do unfortunately.

Reply:
I thought we could override those. Not an issue though. 👍